src/Theme.php

```diff
@@ -219,6 +226,35 @@ public function getParent(): ?Theme
         return $this->parent;
     }
 
+
+    /**
+     * Set theme screenshot.
+     */
+
+    public function setScreenshot(string $screenshot): self
+    {
+
+        $this->screenshot = $screenshot;
+
+        return $this;
+    }
+
+    public function getScreenshotName(): string|null
+    {
+        return $this->screenshot;
+    }
+
+    public function getScreenshotImage(): string
+    {
+        $screenshotImage = $this->getPath() . '/' . $this->screenshot;
```

Comment:
Wouldn't it be better to save the image into the assets path so image can be accessed via url (using `theme_asset` helper in blade) in addition to retrieve it in base64 format.
Then you can directly use `$this->getAssetsPath($this->screenshot);`. to get the full path.


For what I understand the screenshot property of the `composer.json` file should be the image filename (+ extension) only right ?
If so, can you please just add a comment to specify it ? This way I won't forget it when updating the documentation.

Reply:
Hi @gaetan-hexadog I have a small question. If we use URL instead of base64 to display the image, we will pass the screenshot image to the public folder along with CSS/JS/image, right?
![image](https://github.com/hexadog/laravel-themes-manager/assets/12608657/d670c91e-9c32-4786-9ced-3d28b917665c)

Sorry for my English

Reply:
I think we can keep both : base64 encoded image and url
You're right, we will pass the screenshot image to the public folder along with CSS/JS/image.

It's a great idea to provide a default screenshot 💯

Reply:
@gaetan-hexadog thank for you suggestion! I updated. :D